docs/modules/ROOT/pages/how-to/use-tilt.adoc

```diff
@@ -0,0 +1,201 @@
+--
+
+. Click on the link above to open it on your browser.
+. Click "Display token" and copy the login command shown in "Log in with this token"
+. Paste the `oc login` command on the terminal:
++
+[source,shell]
+--
+$ oc login --token=sha256~_xxxxxx_xxxxxxxxxxxxxxxxxxxxxx-xxxxxxxxxx-X --server=https://api.[YOUR_CHOSEN_ZONE].appuio.cloud:6443
+--
+
+. Create a new project called "[YOUR_USERNAME]-fortune-go"
++
+[source,shell]
+--
+$ oc new-project [YOUR_USERNAME]-fortune-go
+Now using project "[YOUR_USERNAME]-fortune-go" on server "https://api.[YOUR_CHOSEN_ZONE].appuio.cloud:6443".
+--
+
+=== 2. Login to the {product} Container Registry
+
+Tilt uses Docker to build the container image, and to publish to a container registry. In this example we're going to use the {product} integrated container registry.
+
+. Login to the {product} container registry (as explained in xref:how-to/use-integrated-registry.adoc[this page]):
++
+[source,shell]
+--
+$ oc whoami -t | docker login registry.${zone}.appuio.cloud -u $(oc whoami) --password-stdin
+--
+
+=== 3. Clone the sample "Fortune in Go" project
+
+The "Fortune in Go" project provides a very simple Go application to start with. You can clone it from GitLab:
+
+[source,shell]
+--
+$ git clone https://gitlab.com/vshn/applications/fortune-go.git
+$ cd fortune-go
+--
+
+=== 4. Add a Deployment File
+
+Add a file named `deployment.yaml` at the root of the project with the following information:
+
+[source,yaml]
+--
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: fortune-go
+spec:
+  template:
+    spec:
+      containers: # <1>
+      - image: registry.[YOUR_CHOSEN_ZONE].appuio.cloud/fortune-go/fortune-go:latest
+        imagePullPolicy: Always
+        name: fortune-container
+        ports:
+        - containerPort: 8080
+    metadata:
+      labels:
+        app: fortune-go
+  selector:
+    matchLabels:
+      app: fortune-go
+  strategy:
+    type: Recreate
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: fortune-go
+spec:
+  ports:
+    - port: 8080
+      targetPort: 8080
+  selector:
+    app: fortune-go
+  type: ClusterIP
+---
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  annotations:
+    cert-manager.io/cluster-issuer: letsencrypt-production
+  name: fortune-go-ingress
+spec:
+  rules:
+  - host: [YOUR_USERNAME]-fortune-go.apps.[YOUR_CHOSEN_ZONE].appuio.cloud # <1>
+    http:
+      paths:
+      - pathType: Prefix
+        path: /
+        backend:
+          service:
+            name: fortune-go
+            port:
+              number: 8080
+  tls:
+  - hosts:
+    - [YOUR_USERNAME]-fortune-go.apps.[YOUR_CHOSEN_ZONE].appuio.cloud
+    secretName: fortune-go-cert
+--
+<1> Remember to customize the parts marked as `[YOUR_USERNAME]` and `[YOUR_CHOSEN_ZONE]` to your liking (and according to the xref:references/zones.adoc[Zones documentation page]).
```

Comment:
Ah, here finally it is, that link to the zone list. This should most certainly appear earlier in the document. It probably won't hurt to have it more than once.

Reply:
Added a link to that page above as well